app/views/payment-links/edit-reference.njk

```diff
@@ -100,6 +100,9 @@
                 classes: "govuk-fieldset__legend--l"
             }
         },
+        hint: {
+          text: "You can use numbers or words in your payment reference. For example, you can include the applicant's name or an existing reference number."
```

Comment:
Can you use a curly apostrophe here?

Like this one → ’

Reply:
Done thanks